### DISCUSSION TITLE
Rebranding

### DISCUSSION THREAD

**Discussion by SoongNoonien** [SoongNoonien/mpdevil#94] [Show and tell]
Hi!
I'm currently working on a rebranding for the player which would be applied with the upcoming GTK4 release. The name and icon don't really comply with the [GNOME Hig](https://developer.gnome.org/hig/) or the new [Flathub quality guidelines](https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines/).
I have thought about that a lot and the best name I came up with is "Record Album" or possibly in German "Plattenalbum". Sadly the intended meaning may not be very well know. A record album or at least a Plattenalbum is a book like object with transparent sleeves instead of normal paper pages which are used to store vinyls (mostly singles) in a compact way. "Record Binder" may be another word for the same thing.
This can easily be paired with an icon resembling exactly such a record album:
![Vorschau6](https://github.com/SoongNoonien/mpdevil/assets/49582808/cdab1268-8f9f-4cb3-9723-1c65732fbf7e)

What do you think about it?

**Comment by bonelifer** [comment]
As someone whose only language is English, I rather like Plattenalbum. As "Record Album" is so generic, I'd pass it by in a app search. Plattenalbum along with the icon, seems like a good pairing.

**Reply by SoongNoonien** [reply]
Ok, thanks for your thoughts! Plattenalbum is a rather old and colloquial expressions, so it is not even that generic in German. This could be an advantage as well.

**Comment by ghost** [comment]
I prefer Plattenalbum to Record Album as well, my only question is whether it's functionally any more descriptive than mpdevil ?
I've spent quite a bit of time trying (and failed) to think of something better so +1 for Platternalbum 

I understand the reasons to replace the icon but I'm going to miss the original mpdevil icon!!

**Reply by SoongNoonien** [reply]
> I prefer Plattenalbum to Record Album as well, my only question is whether it's functionally any more descriptive than mpdevil ?

Yes, I would say so. At least "album" should be universally known and related to music to some degree while mpdevil is just a combination of letters. In the worst case people didn't even read it as mp-devil (Music-Player-Devil)  but as mpd-evil.

**Comment by Clewsy** [comment]
Another +1 vote for Plattenalbum. :)

**Comment by rien333** [comment]
Was not really into the default icon, so I like the change. I'm fine with the new name, though while I collected vinyls during my teens (10 years ago), I've never heard of the concept of an "Plattenalbum". Whether that is a problem may be a different question, however.

**Reply by SoongNoonien** [reply]
> I've never heard of the concept of an "Plattenalbum".

Yes, the concept is quite old. In Germany they where popular around the 50s and 60s to store single and sometimes LPs. But I'm not completely sure about the exact time because I was not around back then ;-).

> Whether that is a problem may be a different question, however.

I don't think that this is a huge problem. Take look at [Amberol](https://apps.gnome.org/Amberol/) for example, the referenced object is even older there.

**Comment by SoongNoonien** [comment]
Thanks again for your feedback! I've now changed the name. Some traces of mpdevil will remain for a bit longer until I make the GTK4 release and change the repo name. Changing the repo name should be relatively harmless as the old links will remain intact.
